pkg/tsdb/stackdriver/stackdriver.go

```diff
@@ -465,14 +485,20 @@ func (e *StackdriverExecutor) parseResponse(queryRes *tsdb.QueryResult, data Sta
 		}
 	}
 
-	queryRes.Meta.Set("resourceLabels", resourceLabels)
-	queryRes.Meta.Set("metricLabels", metricLabels)
+	queryRes.Meta.Set("labels", labels)
 	queryRes.Meta.Set("groupBys", query.GroupBys)
-	queryRes.Meta.Set("resourceTypes", resourceTypes)
 
 	return nil
 }
 
+func toSnakeCase(str string) string {
+	matchFirstCap := regexp.MustCompile("(.)([A-Z][a-z]+)")
```

Comment:
Did this come from [here](https://www.golangprograms.com/golang-convert-string-into-snake-case.html)? I think that code guards against the initial capital letter and against runs of capital letters, which shouldn't happen in the API JSON response. Just curious, why wouldn't the following have sufficed?
```go
func toSnakeCase(str string) string {
	matchAllCap := regexp.MustCompile("(.)([A-Z][a-z]*)")
	return strings.ToLower(matchAllCap.ReplaceAllString(str, "${1}_${2}"))
}
```

Reply:
That's is sufficient, thank you!